src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -186,6 +186,8 @@ export default {
 			boundaryElement: document.querySelector('.main-view'),
 			isQualityWarningTooltipDismissed: false,
 			mouseover: false,
+			audioEnabledBeforeSpacebarKeydown: undefined,
+			spacebarKeyDown: false,
```

Comment:
(Independently of the main review) I would rename `spacebarKeyDown` to something more generic like `isPushToTalkActive`.